astroid/nodes/scoped_nodes/scoped_nodes.py

```diff
@@ -1979,6 +1979,19 @@ def frame(self: T, *, future: Literal[None, True] = None) -> T:
         """
         return self
 
+    @decorators_mod.cachedproperty
+    def tolineno(self) -> Optional[int]:
+        """The last line that this node appears on in the source code."""
```

Comment:
```suggestion
    def tolineno(self) -> Optional[int]:
        """The last line that this node appears on in the source code.
        
        TODO: Remove when we do not support python 3.7 anymore.
        """
```

Reply:
Not sure that comment makes much sense. The whole `tolineno` logic can be updated / deprecated then.